### ISSUE TITLE
How to evaluate the acc of completed prototypes?

### ISSUE THREAD

**Issue by eric-hang** [zhangbq-research/Prototype_Completion_for_FSL#6]
How to get the results in Fig 4. I just use the logits from https://github.com/zhangbq-research/Prototype_Completion_for_FSL/blob/a8042df2bcf0a7f92a2fb955d0dc43727e245245/models/classification_heads.py#L142 but it seems not work.

I'm looking forward to your reply.

Thanks

**Comment by zhangbq-research** [comment]
Thank you for your attention to our paper. The revised code is right. I guess you didn't rerun the meta-training step (i.e., the fine tuning step on base classes), which cannot be neglected during evaluating the performance of completed prototypes.
